### DISCUSSION TITLE
If no expiration_date is known, would 9999-12-31 be the appropriate value to represent no defined expiration date?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#42] [Q&A] [answered]
In the In-Network Rate schema, the expiration date is noted as required.  If the negotiated rate with a Provider does not have a defined expiration, should we set the value to 9999-12-31?  If not, then how should we reflect this scenario?

**Comment by ghost** [comment, marked as answer]
I agree there are some provider agreements that do not have a expiration date and those contracts tend to use a high date of 9999-12-31 to represent that fact.

**Reply by tdfow** [reply]
We are using 9999-12-31 under this scenario.    If that is the expected date for when no expiration date is known, then thought the schema could indicate that 9999-12-31 equates to no defined expiration date.

**Reply by shaselton-usds** [reply]
@tdfow and @BobSyracuse - Thanks for the flag. We added some documentation here: https://github.com/CMSgov/price-transparency-guide/commit/3ca66505da4cf1072d1b4665b0d78699ab26c0e2

We've also had some questions around what to do with evergreen arrangements and made some clarifications on that front too.
